src/ValueObjects/DataSource.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace App\ValueObjects;
+
+use RuntimeException;
+
+final class DataSource
```

Comment:
I think this one is not needed any more

Reply:
Yes, it is not needed anymore, but what do u think in this case if one of the `NewsProviders` forgot to add validation on the data source that it must exist then always we need to validate the data source right? when u ask for a `DataSource` u get a valid `DataSource`